module/VuFind/tests/integration-tests/src/VuFindTest/Files/BadStringsTest.php

```diff
@@ -86,7 +86,7 @@ public function testForBadStrings(): void
                 $failures[] = str_replace(APPLICATION_PATH . '/', '', $fileToCheck) . " ($reasonMsg)";
             }
         }
-        $this->assertEquals('', implode(PHP_EOL, $failures), 'Found bad strings in files.');
+        $this->assertEquals('', implode(PHP_EOL . ' ', $failures), 'Found bad strings in files.');
```

Comment:
Let's add a comment to explain why we did this:

```suggestion
        // We could use a variety of assertions here, but the goal is to make actionable information
        // conveniently available. By imploding the list of bad files (with some extra spaces to separate
        // the diff markers from the filenames) we make it easier to read (and in some setups, click on)
        // the list of files that need attention.
        $this->assertEquals('', implode(PHP_EOL . ' ', $failures), 'Found bad strings in files.');
```